landing_page_backend/landing_page_backend/settings.py

```diff
@@ -122,3 +122,10 @@
 
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
+
+EMAIL_HOST = 'smtp.ukr.net'
+EMAIL_PORT = 465
+EMAIL_HOST_USER = 'testsmtpmail1@ukr.net'
+EMAIL_HOST_PASSWORD = 'WNCr2QnsV3dn8CTL'
```

Comment:
Keep secrets, i.e. password, access token in the source code in secure. You should read them from environment. TSD (Two Scoops of Django) recommends using django-environ app to read such data from `.env` files.